### PR TITLE
Create installation_readme_update

### PR DESCRIPTION
**Pull request by WlinkNET**
Update readme file. Full installation instruction

### CODE REVIEW

File: installation_readme_update [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+### **Install Fedora dependencies (if not using Ubuntu/Debian OS)**
+```
+sudo dnf install git pkgconfig automake autoconf ncurses-devel python python-zmq wget gtest-devel gcc gcc-c++ libtool curl patch
+```
+### **Build arrowd**
+
+```
+1. git clone https://github.com/Arrowchain/arrow.git arrow
+2. cd arrow
+3. git checkout arrow-1927
```
Comment (j4ys0n) [inline installation_readme_update]: remove this line

File: installation_readme_update [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+### **Create config file**
+```
+1. mkdir ~/.arrow
+2. echo "rpcuser=username" >> ~/.arrow/arrow.conf
+3. echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >> ~/.arrow/arrow.conf
+4. echo "rpcallowip=127.0.0.1" >> ~/.arrow/arrow.conf
+5. echo "addnode=110.185.105.21:7654" >> ~/.arrow/arrow.conf
+6. echo "addnode=18.204.209.173:7654" >> ~/.arrow/arrow.conf
+7. echo "addnode=110.185.105.21:7654" >> ~/.arrow/arrow.conf
```
Comment (j4ys0n) [inline installation_readme_update]: duplicate, remove

File: installation_readme_update [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+8. echo "addnode=195.10.210.35:7654" >> ~/.arrow/arrow.conf
+9. echo "addnode=80.11.223.136:7654" >> ~/.arrow/arrow.conf
+10. echo "addnode=52.90.76.26:7654" >> ~/.arrow/arrow.conf
+11. echo "addnode=52.90.76.26:7654" >> ~/.arrow/arrow.conf
```
Comment (j4ys0n) [inline installation_readme_update]: duplicate, remove

File: installation_readme_update [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+6. 
+7. brew install gcc5 --without-multilib
+
+```
+### **Build**
+
+```
+1. git clone https://github.com/Arrowchain/arrow.git arrow
+2. cd arrow
+3. git checkout arrow-1927
```
Comment (j4ys0n) [inline installation_readme_update]: remove

File: installation_readme_update [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+2. echo "rpcuser=username" >> ~/Library/Application Support/arrow/arrow.conf
+3. echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >> ~/Library/Application Support/arrow/arrow.conf
+4. echo "rpcallowip=127.0.0.1" >> ~/Library/Application Support/arrow/arrow.conf
+5. echo "addnode=110.185.105.21:7654" >> ~/Library/Application Support/arrow/arrow.conf
+6. echo "addnode=18.204.209.173:7654" >> ~/Library/Application Support/arrow/arrow.conf
+7. echo "addnode=110.185.105.21:7654" >> ~/Library/Application Support/arrow/arrow.conf
+8. echo "addnode=195.10.210.35:7654" >> ~/Library/Application Support/arrow/arrow.conf
+9. echo "addnode=80.11.223.136:7654" >> ~/Library/Application Support/arrow/arrow.conf
+10. echo "addnode=52.90.76.26:7654" >> ~/Library/Application Support/arrow/arrow.conf
+11. echo "addnode=52.90.76.26:7654" >> ~/Library/Application Support/arrow/arrow.conf
```
Comment (j4ys0n) [inline installation_readme_update]: duplicate, remove

File: installation_readme_update [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+### **Create config file**
+```
+1. mkdir ~/Library/Application Support/arrow/
+2. echo "rpcuser=username" >> ~/Library/Application Support/arrow/arrow.conf
+3. echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >> ~/Library/Application Support/arrow/arrow.conf
+4. echo "rpcallowip=127.0.0.1" >> ~/Library/Application Support/arrow/arrow.conf
+5. echo "addnode=110.185.105.21:7654" >> ~/Library/Application Support/arrow/arrow.conf
+6. echo "addnode=18.204.209.173:7654" >> ~/Library/Application Support/arrow/arrow.conf
+7. echo "addnode=110.185.105.21:7654" >> ~/Library/Application Support/arrow/arrow.conf
```
Comment (j4ys0n) [inline installation_readme_update]: duplicate, remove

File: installation_readme_update (hunk 2 above)
Comment (WlinkNET) [reply installation_readme_update]: ok done

File: installation_readme_update (hunk 4 above)
Comment (WlinkNET) [reply installation_readme_update]: ok

File: installation_readme_update (hunk 5 above)
Comment (WlinkNET) [reply installation_readme_update]: ok

File: installation_readme_update (hunk 6 above)
Comment (WlinkNET) [reply installation_readme_update]: ok

**Review by j4ys0n — APPROVED** [review]

**Pull request merged**